src/ledger.ts

```diff
@@ -1643,7 +1643,8 @@ export class LedgerV2SignMultisigTransaction extends LedgerBitcoinV2WithRegistra
   async signPsbt(): Promise<LedgerSignatures[]> {
     return this.withApp(async (app: AppClient) => {
       this.signatures = await app.signPsbt(
-        this.psbt,
+        // unchained-bitcoin mimics ledgers' methods
+        this.psbt as unknown as LedgerPsbtV2,
```

Comment:
Don't think you need the double assertion here

Reply:
You are correct! I was getting an error here at some point but could've been my editor getting screwed up with all the package changes.